src/styles/isomer-cms/elements/buttons.scss

```diff
@@ -92,6 +92,34 @@ button{
     }
   }
 
+  &.green {
+    background: $login-green;
```

Comment:
similarity with `&.blue` means we should probably abstract our some elements into a mixin or have `&.blue` and `&.green` inherit from a common ancestor

Reply:
We can use a mixin for `&.blue` and `&.green` and pass the color in as an argument